### DISCUSSION TITLE
Release 3.2.4 schedule

### DISCUSSION THREAD

**Discussion by GulamY** [vvvv/SVG#881] [Q&A] [answered]
Hello! There have a number of changes merged since the 3.2.3. I wanted to ask if there was any planned date for 3.2.4 release?

**Comment by mrbean-bremen** [comment, marked as answer]
I think we can go for a release as soon as the security issue is fixed.

**Reply by GulamY** [reply]
I guess this is the PR that needs to be finalized for the security issue https://github.com/svg-net/SVG/pull/873. Am I correct?

**Reply by mrbean-bremen** [reply]
Yes, correct.

**Reply by GulamY** [reply]
Thank you. I will mark the question as answered
